src/mouse.js

```diff
@@ -4,7 +4,8 @@ var pointermap = dispatcher.pointermap;
 // radius around touchend that swallows mouse events
 var DEDUP_DIST = 25;
 
-var WHICH_TO_BUTTONS = [0, 1, 4, 2];
+// left, middle, right, back, forward
+var BUTTON_TO_BUTTONS = [1, 4, 2, 8, 16];
```

Comment:
Oh, wait, they did actually implement it, it’s just incomplete for drag and drop.